lifelines/fitters/__init__.py

```diff
@@ -1250,7 +1250,13 @@ def _compute_central_values_of_raw_training_data(self, df, strata=None, name="ba
             return v
 
         else:
-            described = df.describe(include="all")
+            from distutils.version import LooseVersion
+            if LooseVersion(pd.__version__) >= '1.1.0':
```

Comment:
TIL `LooseVersion`

Reply:
One thing is that distutils will be deprecated in Python 3.9/3.10. Might make setuptools a runtime dependency at some point. Haven't been keeping up with this.